scripts/scripts.js

```diff
@@ -176,7 +176,7 @@ async function loadLazy(doc) {
  */
 function loadDelayed() {
   // eslint-disable-next-line import/no-cycle
-  window.setTimeout(() => import('./delayed.js'), 3000);
+  window.setTimeout(() => import('./delayed.js'), 6000);
```

Comment:
I am wondering if we should delay the whole thing or just the part that's impacting the TBT which is the GTM I suppose.

Reply:
BTW, did you get a chance to look at [this conversation](https://cq-dev.slack.com/archives/C9KD0TT6G/p1679054960183479) and if it may be of help. Copying @meryllblanchet , @rofe.

Reply:
> if we should delay the whole thing or just the part that's impacting the TBT which is the GTM I suppose.

Good point. Please review 332a98f which only delays GTM.